Formula/orientdb.rb

```diff
@@ -39,4 +39,10 @@ def install
   def caveats
     "Use `orientdb <start | stop | status>`, `orientdb-console` and `orientdb-gremlin`."
   end
+
+  test do
+    # quick check if file is there
```

Comment:
`system "#{bin}/orientdb", "start"` and in an `ensure` `system "#{bin}/orientdb", "stop"`